### DISCUSSION TITLE
Added alarm function

### DISCUSSION THREAD

**Discussion by downdawn** [fastapi-practices/fastapi_best_architecture#212] [Ideas]
DingDing、WeChat、FeiShu、Email...

**Comment by wu-clan** [comment]
Can you hypothesize some scenarios?

**Reply by downdawn** [reply]
Task completion push, resource shortage warning, major abnormality warning..

**Reply by yshan2028** [reply]
这个倒是不错的想法，但这种都属于消息体系，分层订阅消息和推送消息，可以考虑继承到系统的 websocket 接口中，增加一个站内消息的功能
